### PR TITLE
Add support for TextFields

### PR DESCRIPTION
**Pull request by tylermadsen**
Finally got a chance to test locally this is a simple patch for #28

### CODE REVIEW

File: condensedinlinepanel/static/condensedinlinepanel/src/condensedinlinepanel.tsx [hunk 1]
```diff
@@ -87,7 +87,7 @@ export function init(id: string, options: Options = {}) {
                 sortOrders.push(state.forms[i].position);
             }
 
-            if (sortOrderField instanceof HTMLInputElement) {
+            if (sortOrderField instanceof HTMLInputElement || sortOrderField instanceof HTMLTextAreaElement) {
```
Comment (kaedroho) [inline condensedinlinepanel/static/condensedinlinepanel/src/condensedinlinepanel.tsx]: This field is always a hidden input so checking for ``HTMLTextAreaElement`` is not required here

File: condensedinlinepanel/static/condensedinlinepanel/src/condensedinlinepanel.tsx [hunk 2]
```diff
@@ -105,13 +105,13 @@ export function init(id: string, options: Options = {}) {
             }
         }
 
-        if (deleteField instanceof HTMLInputElement) {
+        if (deleteField instanceof HTMLInputElement || deleteField instanceof HTMLTextAreaElement) {
```
Comment (kaedroho) [inline condensedinlinepanel/static/condensedinlinepanel/src/condensedinlinepanel.tsx]: Ditto

File: condensedinlinepanel/static/condensedinlinepanel/src/condensedinlinepanel.tsx [hunk 3]
```diff
@@ -105,13 +105,13 @@ export function init(id: string, options: Options = {}) {
...
             deleteField.value = JSON.stringify(deletedForms);
         }
     });
 
     // Set initial state
-    if (dataField instanceof HTMLInputElement) {
+    if (dataField instanceof HTMLInputElement || dataField instanceof HTMLTextAreaElement) {
```
Comment (kaedroho) [inline condensedinlinepanel/static/condensedinlinepanel/src/condensedinlinepanel.tsx]: Ditto

File: condensedinlinepanel/static/condensedinlinepanel/src/condensedinlinepanel.tsx [hunk 4]
```diff
@@ -122,7 +122,7 @@ export function init(id: string, options: Options = {}) {
     store.subscribe(() => {
         let state = store.getState() || emptyState();
 
-        if (totalFormsField instanceof HTMLInputElement) {
+        if (totalFormsField instanceof HTMLInputElement || totalFormsField instanceof HTMLTextAreaElement) {
```
Comment (kaedroho) [inline condensedinlinepanel/static/condensedinlinepanel/src/condensedinlinepanel.tsx]: Ditto

File: condensedinlinepanel/static/condensedinlinepanel/src/components/Card.tsx [hunk 5]
```diff
@@ -105,7 +105,7 @@ export class Card extends React.Component<CardProps, CardState> {
         for (let fieldName in this.props.form.fields) {
             let fieldElement = document.getElementById(`${this.props.formPrefix}-${fieldName}`);
 
-            if (fieldElement instanceof HTMLInputElement) {
+            if (fieldElement instanceof HTMLInputElement || fieldElement instanceof HTMLTextAreaElement) {
```
Comment (kaedroho) [inline condensedinlinepanel/static/condensedinlinepanel/src/components/Card.tsx]: Looks like I forgot to add ``|| fieldElement instanceof HTMLSelectElement`` here

**Comment by tylermadsen** [comment]
@kaedroho sorry for the hastiness on my part, i made those fixes and tested locally--looking good 🙂

**Comment by kaedroho** [comment]
Thanks!

**Pull request merged**